app/lib/finder_api.rb

```diff
@@ -46,6 +46,12 @@ def augment_content_item_with_results(content_item, search_response)
     content_item.details.results = search_response.fetch("results")
     content_item.details.total_result_count = search_response.fetch("total")
 
+    content_item.details.pagination = build_pagination(
+      content_item.details.default_documents_per_page,
+      search_response.fetch('start'),
+      search_response.fetch('total')
+    )
```

Comment:
Previously this was the case, we were passing about a bunch of different things in order to build a Finder but it got hella confusing. It was decided a while ago (https://github.com/alphagov/finder-frontend/pull/215) that instead we should fetch the Content Item, use that to build a search query, use the results of the search query to modify the Content Item and then passed this modified Content Item about.